modules/formulize/templates/admin/element_type_colorpick.html

```diff
@@ -0,0 +1,11 @@
+<div class="panel-content content">
+	<fieldset>
+			<legend>Default Color</legend>
+			<div class="description">
+					<p>Select the default color.</p>
```

Comment:
Not sure if we want something more detailed here?

Reply:
Or anything at all? "Default Color" kind of says it all? But should it be "Colour" ? Formulize, made in Canada.